Sources/Plasma/PubUtilLib/plNetClient/plNetCliAgeJoiner.cpp

```diff
@@ -212,7 +212,7 @@ void plNCAgeJoiner::Start () {
 
     plNetClientMgr * nc = plNetClientMgr::GetInstance();
     nc->SetFlagsBit(plNetClientMgr::kPlayingGame, false);
-    nc->fServerTimeOffset               = 0;    // reset since we're connecting to a new server
+    nc->ResetServerTimeOffset();                // reset since we're connecting to a new server
```

Comment:
Since this doesn't align with anything...

```suggestion
    nc->ResetServerTimeOffset(); // reset since we're connecting to a new server
```